test/test.js

```diff
@@ -1,738 +1,739 @@
 /*jslint node: true, continue: true, eqeq: true, forin: true, nomen: true, plusplus: true, todo: true, vars: true, white: true */
 
-var assert = require("assert");
-var nacl = require("tweetnacl");
-var macaroon = require("../macaroon");
+var assert = require('assert');
+var nacl = require('tweetnacl');
+var macaroon = require('../macaroon');
 
-"use strict";
+'use strict';
 
 function strUint8Array(s) {
-	return nacl.util.decodeUTF8(s);
+  return nacl.util.decodeUTF8(s);
 }
 
 function Uint8ArrayToHex(ua) {
-	if(!(ua instanceof Uint8Array)){
-		throw new Error("invalid Uint8Array:" + ua);
-	}
-	var hex = '';
-	for ( var i = 0; i < ua.length; i++ )
-		hex += (ua[i] < 16 ? '0' : '') + ua[i].toString(16);
-	return hex;
+  if (!(ua instanceof Uint8Array)) {
```

Comment:
FWIW I'm not keen on two space indentation because I find the indent levels easy to confuse with that small an indentation width.